packages/components/config/ember-cli-update.json

```diff
@@ -3,7 +3,7 @@
   "packages": [
     {
       "name": "ember-cli",
-      "version": "5.9.0",
+      "version": "5.12.0",
```

Comment:
this config is wrong it should be updated to similar to this https://github.com/hashicorp/cloud-ui/blob/main/addons/vault-common/config/ember-cli-update.json
and if you are updating it you should run `ember-cli-update` to update the blueprint as well same goes for the apps unless you are planning to do follow up PR

Reply:
Thanks for the feedback. These changes across the packages were made using `ember-cli-update`. If there is something about the changes that makes that not seem the case, please let me know. 

Also in terms of the formatting of the file, can you clarify what alignments you would like to see? Should they have a different format because of the v2 addon conversion?

Reply:
this config is most likely remained unchanged since hds migrated to v2 addon, but v2 addon follows it's own blueprint, so this whole config needs to be updated to similar format I linked above